Mage/src/main/java/mage/abilities/AbilityImpl.java

```diff
@@ -72,6 +72,7 @@
     protected List<Hint> hints = new ArrayList<>();
     protected Outcome customOutcome = null; // uses for AI decisions instead effects
     protected MageIdentifier identifier; // used to identify specific ability (e.g. to match with corresponding watcher)
+    protected String appendToRule = null;
```

Comment:
Miss copy in the copy constructor: `AbilityImpl(final AbilityImpl ability)`

If you add any new fields to xmage's objects then check copy constructor all the time (lifehack: search another field in the same file to find all copy code).

Reply:
Right.  I usually do that.  Just missed it this time.  While I was in there, my IDE was warning me about another variable missing in the copy constructor as well.  Fixed in c5dc43630fe976800e2251c45badf55b7410ff1c